src/editor/components/scenegraph/Toolbar.js

```diff
@@ -55,51 +48,23 @@ export default class Toolbar extends Component {
   }
 
   componentDidUpdate(prevProps) {
```

Comment:
cleaned up componentdidupdate. only calls `cloudSaveHandlerWithImageUpload` if pendingSave is true and there is a currentUser